config/routes.rb

```diff
@@ -230,7 +232,6 @@
     get '/two_factor_options', to: redirect('/authentication_methods_setup')
     patch '/two_factor_options' => 'users/two_factor_authentication_setup#create'
     get '/second_mfa_setup' => 'users/mfa_selection#index'
-    get '/second_mfa_setup/non_restricted' => 'users/mfa_selection#non_restricted'
```

Comment:
I think I'm tracking the changes right, but this route isn't used currently (in production) so it's fine to remove without backwards compatibility?

Reply:
Yes this change isnt used in prod so should be fine.